### DISCUSSION TITLE
Is it possible to get a combination of ScrollArea/TextArea

### DISCUSSION THREAD

**Discussion by Kayrim** [mantinedev/mantine#4620] [Feature requests]
Firstly, love this project and grateful for the work put into.

I wondered if it was possible to get the ScrollArea API incorporated into the TextArea component so we could have more customisation of the scroll.

Similar to the example on the docs:
![image](https://github.com/mantinedev/mantine/assets/57881168/83f25595-e40b-4fdd-ab70-0099878b2453)

Currently working around it like so:

![image](https://github.com/mantinedev/mantine/assets/57881168/09da8f54-a5fe-4558-ae08-e8fd6150b151)

RTE Wrapped in it for now but would be awesome to just have it working with a normal textarea

**Comment by rtivital** [comment]
I doubt that it is possible, ScrollArea consists of multiple elements: root, viewport, scrollbars and textarea does not allow children

**Reply by Kayrim** [reply]
No worries, cant have everything I guess. 👍🏽
